packages/js/src/values/defaultReplaceVariables.js

```diff
@@ -100,5 +100,15 @@ export default function getDefaultReplacementVariables() {
 			label: __( "Current year", "wordpress-seo" ),
 			value: "",
 		},
+		{
+			name: "currentmonth",
+			label: __( "Current month", "wordpress-seo" ),
+			value: "",
+		},
+		{
+			name: "currentday",
+			label: __( "Current day", "wordpress-seo" ),
+			value: "",
+		},
```

Comment:
@Xyfi Is there any particular reason we don't add defaults for the other variables we're introducing, like for currentdate, post_year, author_first_name, etc?

Reply:
As per @Xyfi 's explanation, we added those here because they are not post-specific. So, he also added defaults for currentdate and permalink also. Resolved.

Reply:
On a second look, adding those there will make them appearing in term edit posts, so we decided to remove all the new fields from here.